fpmsyncd/routesync.cpp

```diff
@@ -25,34 +25,32 @@ void RouteSync::onMsg(int nlmsg_type, struct nl_object *obj)
 {
     struct rtnl_route *route_obj = (struct rtnl_route *)obj;
     struct nl_addr *dip;
-    char ifname[MAX_ADDR_SIZE + 1] = {0};
-    uint32_t ipv4;
-    int prefix;
+    char destipprefix[MAX_ADDR_SIZE + 1] = {0};
 
     dip = rtnl_route_get_dst(route_obj);
-    /* Supports IPv4 address only for now */
-    if (rtnl_route_get_family(route_obj)  != AF_INET)
+    nl_addr2str(dip, destipprefix, MAX_ADDR_SIZE);
+    SWSS_LOG_DEBUG("destipprefix=%s\n", destipprefix);
+    /* Supports IPv4 or IPv6 address, otherwise return immediately */
+    switch (rtnl_route_get_family(route_obj))
     {
-        nl_addr2str(dip, ifname, MAX_ADDR_SIZE);
-        SWSS_LOG_INFO("%s: Unknown route family support: %s (object: %s)\n",
-                      __FUNCTION__, ifname, nl_object_get_type(obj));
-        return;
+        case AF_INET:
+        case AF_INET6:
+            break;
+        default:
+            SWSS_LOG_INFO("%s: Unknown route family support: %s (object: %s)\n",
+                          __FUNCTION__, destipprefix, nl_object_get_type(obj));
+            return;
     }
```

Comment:
change this to if (family != AF_NET and family != AF_INET) return. it's more clear then switch.